src/platform/web/ui/general/types.ts

```diff
@@ -13,15 +13,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+export interface IMountArgs {
+    // if true, the parent will call update() rather than the view updating itself by binding to a data source.
+    parentProvidesUpdates: boolean
+};
 
-import {errorToDOM} from "./error.js";
-
-export function mountView(view, mountArgs = undefined) {
-    let node;
-    try {
-        node = view.mount(mountArgs);
-    } catch (err) {
-        node = errorToDOM(err);
-    }
-    return node;
-}
\ No newline at end of file
+export interface UIView {
```

Comment:
Should we perhaps stick to the convention of starting our interfaces with `I`? it seems to be in use everywhere else.

Reply:
Have done so in #486 as I have made changes there to the usage of the interface.